src/mimir_config.py

```diff
@@ -17,12 +17,12 @@
 logger = logging.getLogger(__name__)
 
 ROLES = {
-    "overrides_exporter",
-    "query_scheduler",
+    "overrides-exporter",
```

Comment:
ehm, given the PR description and title, it's not clear to me if this renaming change is intentional? and the alert rule?

Reply:
It's intentional, I updated the PR description :)